api/AudioTrack.json

```diff
@@ -93,7 +93,7 @@
             "version_added": false
           },
           "webview_android": {
-            "version_added": "45"
+            "version_added": false
```

Comment:
Are you sure about this? It's unlisted in [not-webview-exposed.txt](android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt). In six years, I would think someone would notice the false test failures.

Reply:
I'm pretty sure, yes. The AudioVideoTracks feature has always been experimental, and in a grep of current Chromium I can't find anything that enables it for any platform from outside runtime_enabled_features.json5, which is what would have to happen in order to have a difference between platforms.

My understanding of not-webview-exposed.txt is that it tracks differences between WebView and some other config. I'm not sure if the test runs with experimental features enabled or not, but in any case there would be no *difference*, between the configs, the APIs would either be present in both or absent in both.

The effective status of all Chromium browsers should be false, it's just made hard to see by all the flag data.

(I edited your comment to fix the link.)